packages/client/src/runtime/utils/serializeRawParameters.ts

```diff
@@ -43,7 +43,17 @@ function encodeParameter(parameter: any, objectSerialization: 'fast' | 'slow'):
     }
   }
 
-  if (isArrayBufferLike(parameter) || ArrayBuffer.isView(parameter)) {
+  if (ArrayBuffer.isView(parameter)) {
```

Comment:
Something that was not caught before, I propose this solution. Not an expert on DataView, so feel free to suggest changes here.

Reply:
(cc @aqrln)